src/prettify.js

```diff
@@ -159,17 +159,19 @@ export const prettify = (html, config) => {
   const ignore = validated_config.ignore.length > 0
   trim = validated_config.trim
 
+  const protectionString = config?.protection_string || '_!i-£___£%_';
```

Comment:
Can you assign the string to CONFIG in constants.js? At that point we won't need this line.